docs/data/date-pickers/date-picker/CustomizationExamples.js

```diff
@@ -0,0 +1,58 @@
+import * as React from 'react';
+import { AdapterDayjs } from '@mui/x-date-pickers/AdapterDayjs';
+import ToggleButton from '@mui/material/ToggleButton';
+import ToggleButtonGroup from '@mui/material/ToggleButtonGroup';
+import Stack from '@mui/material/Stack';
+import { LocalizationProvider } from '@mui/x-date-pickers/LocalizationProvider';
+import BrandingProvider from 'docs/src/BrandingProvider';
+import CustomizationPlayground from 'docsx/src/modules/components/CustomizationPlayground';
+import CircularProgress from '@mui/material/CircularProgress';
+import { pickerExamples } from './examplesConfig';
+
+export default function CustomizationExamples() {
+  const [selectedPicker, setSelectedPicker] = React.useState(0);
+
+  const handleSelectedPickerChange = (_e, newValue) => {
```

Comment:
```suggestion
  const handleSelectedPickerChange = (_e, newValue) => {
    if (newValue != null) {
     ....
```

Otherwise, cliking again on the selected pickers will unselect it and crash the demo